.github/workflows/cypress_integration.yml

```diff
@@ -0,0 +1,52 @@
+name: Cypress Integration Tests
+
+on:
+  push:
+
+jobs:
+  cypress:
+    runs-on: ubuntu-latest
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
+    env:
+      MAILING_DATABASE_URL_TEST: postgresql://postgres:postgres@localhost:5432/mailing_cli_test
+      WEB_DATABASE_URL_TEST: postgresql://postgres:postgres@localhost:5432/mailing_test
+      MAILING_WEB_SESSION_PASSWORD: test-YEAed65EasPz7zbGF4fuoMTDka7C
+      MAILING_SESSION_PASSWORD: test-5toTch6whgDvErN1nPg0hbDq6Do6
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "yarn"
+      - name: Install dependencies
+        run: yarn
+      - name: Build the app
+        run: yarn build
+      - name: Cypress run integration tests
+        uses: cypress-io/github-action@v4
+        run: test:integration:cypress
+        # env:
+        #   DEBUG: "@cypress/github-action"
+        # with:
+        #   install: false
+        #   start: yarn ci:server:nohup
+        #   working-directory: packages/cli
```

Comment:
if the config is supposed to look more like this (from the regular cypress config), then I think we just have to specify a different start command and the other cypress config

Reply:
Seems like it's working without the cypress action, seems chill 👍